Troika/Extensions/Layout.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+public extension UIView {
+
+    /// Layouts the current view to fit it's superview.
+    ///
+    /// - Parameters:
+    ///   - insets: The inset for fitting the superview.
+    ///   - isActive: A boolean on whether the constraint is active or not.
+    /// - Returns: The added constraints.
+    @discardableResult
+    func fillInSuperview(insets: Insets = .zero, isActive: Bool = true) -> [NSLayoutConstraint] {
+        guard let superview = self.superview else {
+            return [NSLayoutConstraint]()
+        }
+
+        translatesAutoresizingMaskIntoConstraints = false
+
+        var constraints = [NSLayoutConstraint]()
+        constraints.append(topAnchor.constraint(equalTo: superview.topAnchor, constant: insets.top))
+        constraints.append(leadingAnchor.constraint(equalTo: superview.leadingAnchor, constant: insets.leading))
+        constraints.append(bottomAnchor.constraint(equalTo: superview.bottomAnchor, constant: insets.bottom))
```

Comment:
Now that `Insets` always has positive values this will cause the setting of the constraint constant here to have the view bleed outside the superview. The trailing and bottom constraint constant must be negative to have an inset from the superview edges.

Reply:
Duh! You’re right. I’ll fix it.